### ISSUE TITLE
DetailsList - NVDA - Disabled columns are not being read

### ISSUE THREAD

**Issue by eveleens** [OfficeDev/office-ui-fabric-react#10009]
## Codepen demonstrating the issue

https://codepen.io/anon/pen/EqXNxE 

## Actual behavior:
Disabled column headers aren't focusable and aren't being read in NVDA. (Does NVDA only follow elements that are focusable?)

## Expected behavior:
NVDA should be reading column headers regardless.

NVDA should announce row column information of table. When nvda focus is on table, it should announce it as "table having 1 row and 4 columns ".

## Documentation describing expected behavior
This seems like only an issue in NVDA. It works in Narrator.

**Comment by marygans** [comment]
Hi @eveleens! Thanks for submitting this isuse, DetailsList seems to works fine with Narrator/Edge when I test it. Looking back on some previous issues submitted in the past, this is a bug that might be in the works of getting fixed on their end, specifically with NVDA?

**Comment by eveleens** [comment]
Can you link me to some previous issues?

**Comment by marygans** [comment]
Hi @eveleens, Here is an example: #9887

**Comment by eveleens** [comment]
I could see this being an NDVA issue, but I also want to say that #9887 you linked doesn't have anything similar to this issue other than it's also an NVDA issue. 

That being said, I can file a bug with NVDA for this.
